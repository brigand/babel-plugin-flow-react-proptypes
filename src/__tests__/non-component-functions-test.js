const babel = require('babel-core');
const content = `
/* @flow */

const NotComponent = (x: number, y: number): number => {
  return x + y;
};

`;

it('function-types', () => {
  const res = babel.transform(content, {
    babelrc: false,
    presets: ['env', 'stage-1', 'react'],
    plugins: ['syntax-flow', require('../')],
  }).code;
  expect(res.indexOf('prop-types')).toBe(-1);
  expect(res).toMatchSnapshot();
});
