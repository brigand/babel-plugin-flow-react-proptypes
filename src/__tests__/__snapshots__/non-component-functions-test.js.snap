// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`function-types 1`] = `
"\\"use strict\\";

var NotComponent = function NotComponent(x, y) {
  return x + y;
};"
`;
